### PR TITLE
Mobile 4458

### PR DESCRIPTION
**Pull request by crazyserver**

### CODE REVIEW

**Review by NoelDeMartin — CHANGES_REQUESTED** [review]

File: src/core/components/empty-box/core-empty-box.html [hunk 1]
```diff
@@ -1,4 +1,7 @@
-<img *ngIf="image && !icon" [src]="image" role="presentation" alt="">
-<ion-icon *ngIf="icon" [name]="icon" aria-hidden="true" />
+@if (icon) {
+<ion-icon [name]="icon" aria-hidden="true" />
+} @else if (image) {
+<img [src]="image" role="presentation" alt="">
+}
```
[the remark this answers is not in the document]
Comment (crazyserver) [reply src/core/components/empty-box/core-empty-box.html:5]: Yep Auto-formatter is doing that, but I don't know why

File: src/core/components/empty-box/empty-box.ts [hunk 2]
```diff
@@ -25,15 +25,18 @@ import { Component, HostBinding, Input } from '@angular/core';
...
 })
 export class CoreEmptyBoxComponent {
 
     @Input() message = ''; // Message to display.
     @Input() dimmed = false; // Wether the box is dimmed or not.
     @Input() icon?: string; // Name of the icon to use.
     @Input() image?: string; // Image source. If an icon is provided, image won't be used.
-    @Input() flipIconRtl = false; // Whether to flip the icon in RTL. Defaults to false.
+    /**
+     * @deprecated since 4.4. Not used anymore.
```
[the remark this answers is not in the document]
Comment (crazyserver) [reply src/core/components/empty-box/empty-box.ts:37]: I dont know. It was not used so I decided to deprecated now.

File: src/core/components/message/message.scss [hunk 3]
```diff
@@ -34,8 +34,8 @@
         overflow: visible;
 
         &:hover {
-            -webkit-filter: drop-shadow(2px 2px 2px rgba(0,0,0,.3));
-            filter: drop-shadow(2px 2px 2px rgba(0,0,0,.3));
+            -webkit-filter: drop-shadow(2px 2px 2px rgb(0 0 0 / 30%));
+            filter: drop-shadow(2px 2px 2px rgb(0 0 0 / 30%));
```
[the remark this answers is not in the document]
Comment (crazyserver) [reply src/core/components/message/message.scss:38]: By requirement of Ioni7, browserslist version says now 79

File: src/addons/block/activitymodules/components/activitymodules/activitymodules.ts [hunk 4]
```diff
@@ -93,6 +95,8 @@ export class AddonBlockActivityModulesComponent extends CoreBlockBaseComponent i
                     modFullNames[mod.modname] = mod.modplural;
                 }
 
+                brandedIcons[mod.modname]= mod.branded;
```
[the remark this answers is not in the document]
Comment (crazyserver) [reply src/addons/block/activitymodules/components/activitymodules/activitymodules.ts]: Thanks!

File: src/core/components/empty-box/empty-box.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (NoelDeMartin) [reply src/core/components/empty-box/empty-box.ts:37]: It seems like this was removed in 3.9.5 in this commit: https://github.com/moodlehq/moodleapp/commit/5ad0326910f8a6a7d11e329a88882a59ad68227d

So I think we can remove it directly without any deprecation notice.

**Review by NoelDeMartin — CHANGES_REQUESTED** [review]
The only thing left is to remove the code deprecated in 3.9.5 and I'll proceed with the integration.

File: src/core/components/empty-box/empty-box.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (crazyserver) [reply src/core/components/empty-box/empty-box.ts:37]: I prefer to leave the deprecation notice so plugins that have not been notified have time to change it.

**Review by NoelDeMartin — APPROVED** [review]

**Pull request merged**